scapy/contrib/iec104_fields.py

```diff
@@ -0,0 +1,113 @@
+"""
```

Comment:
Please add a short license header

Reply:
done here: cf3ea66

used the format as seen in many other files of the project (therefore changes in more than one file)